src/components/ImagesCarousel.js

```diff
@@ -38,6 +38,8 @@ export const ImagesCarousel = ({
   const [isPaused, setIsPaused] = useState(false);
   const [carouselImageIndex, setCarouselImageIndex] = useState(0);
 
+  const shouldShowPauseButton = showSliderPauseButton && !isDisturber;
```

Comment:
you can do this in line 33 i guess instead of creating a new const here

Reply:
I don't think I can integrate it here. Because if `showSliderPauseButton` returns true from the server, `isDisturber` doesn't matter. Or am I wrong?